sources/functions/gcc

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Functions to upgrade GCC on Ubuntu
+# Stickz for Swizzin 2023
+# GPLv3 Applies
+
+Upgrade_GCC_Ubuntu() {
+    echo_info "Checking OS for GCC upgrade"
+    if [[ "$(_os_codename)" = "focal" ]]; then
+        Upgrade_GCC10_Focal
+    elif [[ "$(_os_codename)" = "jammy" ]]; then
+        Upgrade_GCC12_Jammy
+    else
+        echo_info "Skipping GCC upgrade due to unsupported OS"
```

Comment:
I don't see a reason to inform the user this was skipped. I would suggest an `echo_log_only` here